package.json

```diff
@@ -52,6 +52,7 @@
     "webpack": "^1.8.3"
   },
   "dependencies": {
+    "moment": "^2.10.2",
```

Comment:
The traditional example I always think of is:

_Is this a dependency for continuing development of the thing (e.g. writing and running tests)?_ 
`devDependencies`

_Is this only a dependency for "running" the thing?_
`dependencies`

I'd love to discuss the method of distinction though, because I often encounter contradictions to this in the wild.